pkg/ansible/proxy/proxy.go

```diff
@@ -235,8 +272,12 @@ func Run(done chan error, o Options) error {
 		o.Cache = informerCache
 	}
 
+	if o.ControllerMap == nil {
+		o.ControllerMap = aoController.NewControllerMap()
```

Comment:
I think we may want to error here at least log this is happening because this will behave poorly I think.